contrib/elasticsearch/es7-upgrade/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM python:3.8
+COPY . .
+RUN pip install --upgrade pip
+RUN pip install elasticsearch
+ENTRYPOINT ["python", "transfer.py"]
```

Comment:
How will this work with no args?

Reply:
The args are inputted during docker run so like
`docker build -t migrate-es-7 .`
`docker run migrate-es-7 -s SOURCE -d DEST`
Maybe I should add this into the doc

Reply:
Yes please :)

Reply:
Done!